doc/developer-guides/hld/hv-partitionmode.rst

```diff
@@ -0,0 +1,358 @@
+.. _partition-mode-hld:
+
+Partition mode
+##############
+
+ACRN is type-1 hypervisor that supports running multiple guest operating
+systems (OS). Typically, the platform BIOS/boot-loader boots ACRN, and
+ACRN loads single or multiple guest OSes. Refer to :ref:`hv-startup` for
+details on the start-up flow of the ACRN hypervisor.
+
+ACRN supports two modes of operation: Sharing mode and Partition mode.
+This document describes ACRN's high-level design for Partition mode
+support.
+
+.. contents::
+   :depth: 2
+   :local:
+
+Introduction
+************
+
+ACRN, in partition mode, is used to provide guests with exclusive access
+to cores, memory, cache, and IO devices. There is no resource sharing
+between the guests. ACRN expects static partitioning, by developers, of
```

Comment:
'No resource sharing between the guests' is a stronger claim than what we currently support. The last level cache, memory bandwidth and peripheral bus are still shared. Can we make this statement more specific to what we partition?

Reply:
Sure, will add more details to it.